FutureNNAimbot/Program.cs

```diff
@@ -340,47 +381,52 @@ static void Main(string[] args)
                                 break;
 
                             }
-                            else Console.WriteLine("Then you finished training write \"done\" in this console!");
+                            else Console.WriteLine("When you finished training, write \"done\" in this console.");
                         }
-                        Console.WriteLine("Okey! Training finished :) Let's check detection in the game!");
+                        Console.WriteLine("Okay! Training finished. Let's check detection in the game!");
```

Comment:
Great! Training has finished. Let's now check the detection in "{game}".

Reply:
With "Let's", it implies now, so no need to add now.